# HG changeset patch
# Parent 362268611cc9e381cbeb313dc4c2fdf1f46491d4
# User sbr
# Date 1352231253 -3600

Fix (r24466): Use correct subdir for heightmaps.
When downloading heightmaps from the content window, they were stored
in the 'content_download/scenario' directory instead of in the
'content_download/scenario/heightmap' directory and thus not available
in the "Load Heightmap" window without moving them at hand.

diff -r 362268611cc9 src/network/core/tcp_content.cpp
--- a/src/network/core/tcp_content.cpp	Tue Nov 06 18:45:08 2012 +0000
+++ b/src/network/core/tcp_content.cpp	Tue Nov 06 22:07:17 2012 +0100
@@ -241,9 +241,8 @@
 		case CONTENT_TYPE_BASE_MUSIC:
 			return BASESET_DIR;
 
-		case CONTENT_TYPE_SCENARIO:
-		case CONTENT_TYPE_HEIGHTMAP:
-			return SCENARIO_DIR;
+		case CONTENT_TYPE_SCENARIO:  return SCENARIO_DIR;
+		case CONTENT_TYPE_HEIGHTMAP: return HEIGHTMAP_DIR;
 	}
 }
 #endif /* OPENTTD_MSU */
